src/installer/pkg/projects/netcoreapp/src/netcoreapp.depproj

```diff
@@ -70,6 +70,42 @@
     </ItemGroup>
   </Target>
 
+  <!-- Platform manifest needs to include all System*Native items from 3.1 release. This target adds them explicitly,
+      to work around the issue introduced with a fix for https://github.com/dotnet/runtime/issues/33118,
+      Android libraries need to conform to a specific naming convention which necessitated renames of these libraries.
```

Comment:
(Nit) Extra clarity

```suggestion
  <!-- Platform manifest needs to include all System*Native items from 3.1 release. This target adds them explicitly,
      to fill the gap that appeared when the
      names of these libraries were changed in 5.0 to conform to a specific Android naming convention. The library name change was a fix for https://github.com/dotnet/runtime/issues/33118.
```

(Suggestion feature doesn't handle newlines nicely, so couldn't format it.)